### PR TITLE
balloon_check..iommu_enabled: Add guest restriction

### PR DESCRIPTION
**Pull request by huangyum**
ID: 1896612
Signed-off-by: Yumei Huang <yuhuang@redhat.com>

### CODE REVIEW

**Comment by huangyum** [comment]
@yanan-fu Please help review, thanks.

File: qemu/tests/cfg/balloon_check.cfg [hunk 1]
```diff
@@ -60,6 +60,7 @@
         - iommu_enabled:
             only q35
             only balloon_evict_and_enlarge
+            only Windows
```
Comment (yanan-fu) [inline qemu/tests/cfg/balloon_check.cfg:63]: It is better to add a comment about the product bz 1791593

Comment (huangyum) [reply qemu/tests/cfg/balloon_check.cfg:64]: Thanks, added.

**Review by yanan-fu — APPROVED** [review]
Acked-by: Yanan Fu <yfu@redhat.com>

**Pull request merged**